.circleci/config.yml

```diff
@@ -64,12 +110,84 @@ jobs:
       - run:
           name: Invalidate CloudFront caches
           command: aws cloudfront create-invalidation --distribution-id EPTW7F3CB566V --paths "/*"
-          
+  
+
+  integration-tests-cocoapods:
+    macos:
+      xcode: "12.0"
+    working_directory: ~/purchases-ios/
+    shell: /bin/bash --login -o pipefail
+    steps:
+      - checkout
+      
+      - install-gems:
+          directory: "IntegrationTests/CocoapodsIntegration"
+
+      - run:
+          name: Install Pods
+          command: |
+              cd IntegrationTests/CocoapodsIntegration/
+              pod install
+      
+      - scan-and-archive:
+          directory: "IntegrationTests/CocoapodsIntegration"
+      
+  integration-tests-swift-package-manager:
+    macos:
+      xcode: "12.0"
+    working_directory: ~/purchases-ios/
+    shell: /bin/bash --login -o pipefail
+    steps:
+      - checkout
+
+      - install-gems-scan-and-archive:
+          directory: "IntegrationTests/SPMIntegration/"
+      
+  integration-tests-carthage:
+    macos:
+      xcode: "12.0"
+    working_directory: ~/purchases-ios/
+    shell: /bin/bash --login -o pipefail
+    steps:
+      - checkout
+      
+      # Carthage
+      - restore_cache:
+          keys: 
+            - carthage-cache-{{ checksum "Cartfile.resolved" }}
+      - run:
+          name: Carthage Update
```

Comment:
note that this is slightly different from the carthage step we do for regular tests:
- regular tests: `carthage bootstrap` -> get the version of Nimble / OHHTTPStubs from the Cartfile.lock file
- integration tests: `carthage update` -> get the absolute latest version of the dependency, which in this case is `purchases-ios`